python/beeai_framework/agents/bee/agent.py

```diff
@@ -95,7 +95,7 @@ def meta(self) -> AgentMeta:
             extra_description.append(f"Tool ${tool.name}': ${tool.description}.")
 
         return AgentMeta(
-            name="BeeAI",
+            name="Bee",
```

Comment:
This breaks an e2e test, you'll need to update [`test_multi_agents_workflow_agent_delete`](https://github.com/i-am-bee/beeai-framework/blob/83f11b3b02e39883916f97ffe0cf1f95f0442e3a/python/tests/workflows/test_multi_agents.py#L67)

Reply:
fixed